packs/actions/extract-element.json

```diff
@@ -11,7 +11,7 @@
+            "value": "<p>You extract elemental matter from a creature's body to weaken it and take its power for your own. Target a creature within 30 feet that has a trait matching one of your kinetic elements or is made of one of your kinetic elements. The target takes @Damage[(ceil(@actor.level/2))d4[untyped]]{damage} (with no damage type) and becomes susceptible to your impulses, depending on its @Check[type:fortitude|dc:resolve(@actor.system.proficiencies.classDCs.kineticist.dc)] save against your class DC.</p>\n<hr />\n<p><strong>Critical Success</strong> The creature is unaffected</p>\n<p><strong>Success</strong> The creature takes half damage, and you add some of its elemental matter to your kinetic aura. Your impulses bypass any immunity the creature has to their elemental trait or traits, and the target takes a -1 circumstance penalty to its saves and AC against your impulses. If the target normally has a resistance that would apply to damage from one of your impulses, ignore that resistance; if it normally would be immune to that damage type, it instead has resistance equal to its level to damage from the impulse. You can't target a creature with Extract Element if elemental matter you extracted from it is already in your kinetic aura. These effects last for 5 minutes or until your kinetic aura ends, whichever comes first</p>\n<p><strong>Failure</strong> As success, but the creature takes full damage.</p>\n<p><strong>Critical Failure</strong> As failure, but the creature takes double damage.</p>\n<hr />\n<p><strong>Level (+2)</strong> The damage increases by 1d4.</p>"
```

Comment:
Hard to edit on the phone, but did you intend to enclose `{damage}`?

Reply:
Changed